packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -211,64 +201,64 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
                     onLongPress={handleLongPress}
                   />
                 ) : null}
+                {!isLinkPreviewOnly ? (
+                  <Hyperlink
+                    text={message.message}
+                    styles={{
+                      root: [
+                        styles.message,
+                        isAuthor && styles.messageIsAuthor
+                      ],
+                      link: [
+                        styles.message,
+                        styles.link,
+                        isAuthor && styles.messageIsAuthor
+                      ]
+                    }}
+                  />
+                ) : null}
               </View>
-              {!isLinkPreviewOnly ? (
-                <Hyperlink
-                  text={message.message}
-                  styles={{
-                    root: [styles.message, isAuthor && styles.messageIsAuthor],
-                    link: [
-                      styles.message,
-                      styles.link,
-                      isAuthor && styles.messageIsAuthor
-                    ]
-                  }}
+              {message.hasTail ? (
+                <ChatTail
+                  fill={
+                    isAuthor && !isLinkPreviewOnly
+                      ? palette.secondary
+                      : palette.white
```

Comment:
Do these palette items work correctly in all themes?

Reply:
good q

Reply:
Yep it works